### DISCUSSION TITLE
Idea/Question: Packer sync/install/et al in background?

### DISCUSSION THREAD

**Discussion by SiddharthShyniben** [wbthomason/packer.nvim#835] [Ideas]
Right now, packer always opens a popup window when installing. It would be nice to have some sort of way to not open any popup and notify the user once compiled/hide the popup and only show it if there are any errors while installing

**Comment by sdondley** [comment]
See README documentation. You can run in non-interactive mode with a custom config. Also see documentation README, you can trigger a function when the process completes.
